ndbench-cass-plugins/src/main/java/com/netflix/ndbench/plugin/cass/CassJavaDriverPlugin.java

```diff
@@ -17,8 +17,6 @@
 package com.netflix.ndbench.plugin.cass;
 
 import com.datastax.driver.core.*;
```

Comment:
regarding imports... i ran optimize imports.. but by mistake did not limit to vcs changed files. so i changed this file.. '

it was :+1: 
import com.datastax.driver.core.*;
import com.datastax.driver.core.ConsistencyLevel;
import com.datastax.driver.core.Row;

the optimize imports dropped the 2 redundant imports.   If you want those back i will happily put them back.. just let me know !